src/TitleDetails/TitleDetailsContainer.js

```diff
@@ -113,6 +116,18 @@ const TitleDetailsContainer = ({ location, history, mutator, match, resources })
     [fetchReceivingResources, showCallout, titleId],
   );
 
+  useEffect(() => {
+    setLocations();
+
+    if (pieces && poLine.id) {
+      getLocations(mutator.locations, pieces, poLine)
+        .then(setLocations)
+        .catch(() => setLocations([]));
```

Comment:
does it make sense to do this? If you pass no locations, component might be broken in case something was selected